models/POP/README.rst

```diff
@@ -0,0 +1,440 @@
+##########
+POP README
+##########
+
+Contents
+========
+
+#. `Overview`_
+#. `Observations`_
+#. `Namelist`_
+#. `Files`_
+#. `Terms of Use`_
+
+Overview
+========
+
+The **Parallel Ocean Program (POP)** may be used with the **Data Assimilation
+Research Testbed (DART)**. Both main variants - `LANL POP
+<https://climatemodeling.science.energy.gov/projects/climate-ocean-and-sea-ice-modeling-cosim>`_
+and `CESM2.0 POP2 <https://www.cesm.ucar.edu/models/cesm2/ocean/>`_ have been
+tested.
```

Comment:
I've added stuff to the effect that the LANL POP workflow has been tested with LANAI and that the CESM POP workflow is compatible with Manhattan.